### DISCUSSION TITLE
Question: Is mastodon licensed under the GNU AGPL v3 or later or the GNU AGPL v3 only?

### DISCUSSION THREAD

**Discussion by RealJosephKnapp** [mastodon/mastodon#32209] [General]
I wanted to ask if mastodon is licensed under the GNU AGPL v3 or later of the GNU AGPL v3 only. I wanted clarification because it doesn't say in the license, only stating it is licensed under the GNU AGPL v3, but not if later versions of that license are compatible.

**Comment by mjankowski** [comment]
FYI the license itself describes how to handle this - https://github.com/mastodon/mastodon/blob/v4.5.1/LICENSE#L568-L575
